installer/install.sh

```diff
@@ -30,7 +59,7 @@ fi
 
 cd $(dirname $0)
 if [ -r ./machine.conf ]; then
-    . ./machine.conf
+    read_conf_file "./machine.conf"
 fi
 
 if [ -r ./onie-image-arm64.conf ]; then
```

Comment:
>onie-image-arm64.conf [](http://example.com/codeflow?start=10&length=21)

Will you add another block for `onie-image-armhf.conf` ?

Reply:
Yes I will need to. Good catch thanks.

Reply:
Ultimately with these changes I am just looking to make sure the build succeeds and SONiC successfully installs via both ONIE and SONiC-SONiC upgrade / downgrade.